src/crewai/crew.py

```diff
@@ -413,11 +421,15 @@ def _run_hierarchical_process(self) -> Union[str, Dict[str, Any]]:
                 )
 
         self._finish_execution(task_output)
+
         # type: ignore # Incompatible return value type (got "tuple[str, Any]", expected "str")
-        manager_token_usage = manager._token_process.get_summary()
-        return self._format_output(
-            task_output, manager_token_usage
-        ), manager_token_usage
+        manager_token_usage = manager.token_process.get_summary()
+        if hasattr(manager, "token_process"):
```

Comment:
Same. Maybe it's actually a good idea to do that right now, given how simple it would be now.